pkg/cfgdir/cfgdir.go

```diff
@@ -14,9 +14,13 @@ import (
 
 type Config struct {
 	OptOutAnalytics bool `json:"opt_out_analytics"`
+
+	KubeHost    string `json:"kube_host"`
+	ManagerHost string `json:"manager_host"`
+	ManagerCert string `json:"manager_cert"`
 }
 
-var ConfigDir string
+var ConfigDir, KubeHostOverride string
```

Comment:
Setting this as a global var feels pretty sketchy to me, especially since it's set via `ParseConfig` rather than `init`. I'd like to either see the config get parsed in `init` (and nothing else would call `ParseConfig` directly), or have the `KubeHostOverride` be passed to consumers via function args.

Reply:
I needed this for the KubeCert change, so I just edited this patch and made the change to call `ParseConfig` from `up.go`, and merged it.